packages/Checkbox/src/checkbox.scss

```diff
@@ -1,291 +1,367 @@
+    height: .1rem;
+    width: .1rem;
+    overflow: hidden;
 }
 
 .ids-checkbox__box {
-  background: var(--ids-checkbox-background);
-  border: .1rem solid var(--ids-checkbox-border);
-  border-radius: var(--ids-checkbox-completion-radius);
-  box-sizing: border-box;
-  color: var(--ids-checkbox-check);
-  cursor: pointer;
-  display: inline-block;
-  flex: 0 0 auto;
-  height: var(--ids-checkbox-completion-size);
-  margin: 0;
-  overflow: hidden;
-  position: relative;
-  transition: background 0.05s ease-in-out;
-  width: var(--ids-checkbox-completion-size);
-
-  .ids-checkbox--completion:checked ~ .ids-checkbox__label & {
-    border-color: var(--ids-checkbox-border-checked);
-    background: var(--ids-checkbox-background-checked);
-  }
-
-  .ids-checkbox--completion:disabled ~ .ids-checkbox__label & {
-    cursor: not-allowed;
-    border-color: var(--ids-checkbox-completion-border-disabled);
-    background: var(--ids-checkbox-completion-background-disabled);
-  }
-
-  .ids-checkbox--completion:focus ~ .ids-checkbox__label & {
-    outline: none;
-    box-shadow: var(--ids-checkbox-focus);
-  }
-
-  .ids-checkbox--completion:hover ~ .ids-checkbox__label &,
-  .ids-checkbox--completion:checked:hover ~ .ids-checkbox__label & {
-    border-color: var(--ids-checkbox-border-hover);
-  }
-
-  .ids-checkbox--completion:focus:not(:focus-visible) ~ .ids-checkbox__label & {
-    box-shadow: none;
-  }
-
-  .ids-checkbox--completion:disabled:hover ~ .ids-checkbox__label & {
-    border-color: var(--ids-checkbox-border);
-  }
-
-  .ids-checkbox--completion ~ .ids-checkbox__label & .ids-checkbox__check {
-    color: var(--ids-checkbox-completion-check-color-default);
-    font-size: 0;
-    height: var(--ids-checkbox-completion-check-size);
-    left: 50%;
-    position: absolute;
-    stroke-dasharray: 20;
-    stroke-dashoffset: 0;
-    top: 50%;
-    transform: translate(-50%, -50%);
-    width: var(--ids-checkbox-completion-check-size);
-  }
-
-  .ids-checkbox--completion:hover ~ .ids-checkbox__label & .ids-checkbox__check {
-    color: var(--ids-checkbox-completion-check-color-hover);
-  }
-
-  .ids-checkbox--completion:disabled ~ .ids-checkbox__label & .ids-checkbox__check {
-    display: none;
-  }
-
-  .ids-checkbox--completion:checked ~ .ids-checkbox__label & .ids-checkbox__check {
-    animation: check-completion-animation .30s ease-out .15s forwards;
-    color: var(--ids-checkbox-completion-check-color-selected);
-    stroke-dashoffset: 20;
-  }
-
-  .ids-checkbox--completion:checked:hover ~ .ids-checkbox__label & .ids-checkbox__check {
-    color: var(--ids-checkbox-completion-check-color-selected);
-  }
-
-  .ids-checkbox--completion:checked:disabled ~ .ids-checkbox__label & .ids-checkbox__check {
-    animation: none;
-    color: var(--ids-checkbox-completion-check-color-disabled);
-    display: block;
-    stroke-dashoffset: 0;
-  }
+    background: var(--ids-checkbox-background);
+    border: .1rem solid var(--ids-checkbox-border);
```

Comment:
this won't compute the same between both themes as the rem is not on the same base.